From eebf241317505e1c925026211abdaa0d542c35c4 Mon Sep 17 00:00:00 2001
From: l00525341 <liuqi115@huawei.com>
Date: Thu, 11 Nov 2021 16:29:32 +0800
Subject: [PATCH] openeuler_defconfig: Build HISI PMU drivers as modules.
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: eebf241317505e1c925026211abdaa0d542c35c4


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4A1WQ

------------------------------------------------------------------------

Build HISI PMU drivers as modules.

Signed-off-by: Qi Liu <liuqi115@huawei.com>
Reviewed-by: Shaokun Zhang <zhangshaokun@hisilicon.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 7e6902f64c66..76d6a118330d 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -5996,7 +5996,7 @@ CONFIG_QCOM_L3_PMU=y
 CONFIG_THUNDERX2_PMU=m
 CONFIG_XGENE_PMU=y
 CONFIG_ARM_SPE_PMU=y
-CONFIG_HISI_PMU=y
+CONFIG_HISI_PMU=m
 # end of Performance monitor support
 
 CONFIG_RAS=y
-- 
2.26.2

